test/config/yaml.test.ts

```diff
@@ -0,0 +1,23 @@
+import assert from 'node:assert/strict';
+import { execSync } from 'node:child_process';
+import test from 'node:test';
+import { resolve } from '../../src/util/path.js';
+
+const cwd = resolve('fixtures/config/yaml');
+
+const exec = (command: string) => {
+  try {
+    const output = execSync(command.replace(/^knip/, 'node ../../../dist/cli.js'), { cwd });
+    return output.toString().trim();
+  } catch (error) {
+    if (error instanceof Error && 'stdout' in error && Buffer.isBuffer(error['stdout'])) {
+      return error['stdout'].toString();
+    }
+    // Unknown error
+    throw error;
```

Comment:
This helper is great! Looks like it resolves https://github.com/webpro/knip/issues/253 :)

Will be very useful going forward and testing CLI arguments, etc.

This looks like using the same `exec` function in the new test files, do you think it could be a single reusable test helper? There's a `test/helpers` folder for this.

Reply:
Sure thing! Will do that ASAP 😁

Reply:
Done ! I've used a "dumb" way where you need to pass the cli path, this could be enhanced further where the execFactory crawls back up to find the cli itself, but for a first iteration, I opted to keeping it KISS. Let me know what you think !

Reply:
Resolving the path to `cli.js` from the helper seems more ergonomic and clean, so would be great if you could still add that. Happy to merge either way!

Reply:
See the other comment, I've added it in the least intrusive way, since doing it the proper way could have an impact on test speed (fs apis to check for file and read files, for every test that uses the exec util)